src/components/SettlementButton.tsx

```diff
@@ -227,15 +190,15 @@ function SettlementButton({
             anchorAlignment={kycWallAnchorAlignment}
             shouldShowPersonalBankAccountOption={shouldShowPersonalBankAccountOption}
         >
-            {(triggerKYCFlow, buttonRef) => (
+            {(triggerKYCFlow: TriggerKYCFlow, buttonRef: MutableRefObject<null>) => (
```

Comment:
why this ref is typed as `null`?

Reply:
If you track it down it should be `HTMLDivElement | View` instead, it's not typed yet as BaseKYCWall is in JS